webpack.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
-1 on having a separate webpack config file for each module, this should be handled at the repo level (and in a separate PR).

Why do we have so many different build variants?